src/sentry/static/sentry/less/layout.less

```diff
@@ -11,7 +11,6 @@ body {
   -webkit-font-smoothing: antialiased;
   overflow-x: hidden;
   padding-left: @sidebar-width;
-  padding-top: 30px;
```

Comment:
Can you check how the alerts at the top are affected (i.e. when your workers haven't been turned on alert)?

Reply:
good catch, removed those too